host-agent/internal/realtime/client.go

```
@@ -0,0 +1,301 @@
+		ticker := time.NewTicker(20 * time.Second)
+		defer ticker.Stop()
+		for {
+			select {
+			case <-ticker.C:
+				pingTimeout, cancel := context.WithTimeout(pingCtx, 5*time.Second)
+				writeMu.Lock()
+				err := connection.Ping(pingTimeout)
+				writeMu.Unlock()
+				cancel()
+				if err != nil {
+					connection.CloseNow()
```

golangci-lint (2.12.2)

[error] Error return value of `connection.CloseNow` is not checked

(errcheck)